app/pages/resetPasswordSuccess/page.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+import Messages from "./messages";
+
+export default function resetPasswordSucess() {
+  return (
+    <div className="flex justify-center items-center">
+      <div className="flex-1 flex flex-col w-full max-w-md px-8 py-8 gap-2">
+        <>
```

Comment:
Can remove the `<>`…`</>` given you have a `<div>` at the top level.

(Obviously compiles to nothing, but removing it makes the code a little easier to follow.)

Reply:
Good catch will fix that up